docs/MO_DG/prepare_model/Model_Optimizer_FAQ.md

```diff
@@ -2,6 +2,11 @@
 
 @sphinxdirective
 
+.. important::
+
+   This article refers to legacy functionality only.
```

Comment:
What is `legacy functionality`? Where is it defined? Any reference?

Reply:
> What is `legacy functionality`? Where is it defined? Any reference?

**Model Optimizer** is referenced in https://docs.openvino.ai/nightly/openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer.html

Reply:
As agreed with @popovaan 

```suggestion
   All of the issues below refer to :doc:`legacy functionalities <openvino_docs_MO_DG_prepare_model_customize_model_optimizer_Customize_Model_Optimizer>`.
```